Adding maemo specific logger config (as there is no syslog daemon by default).
--- a/src/strongswan.conf
+++ b/src/strongswan.conf
@@ -5,6 +5,21 @@
 	retransmit_timeout = 3.0
 	retransmit_base = 1.4
 
+	# maemo specific logging
+	filelog {
+		/var/log/charon.log {
+			time_format = %b %e %T
+			append = no
+			default = 1
+			flush_line = yes
+		}
+	}
+	# there is no syslog daemon installed by default
+	syslog {
+		daemon {
+		}
+	}
+
 	# number of worker threads in charon
 	threads = 16
 
